File: sqlite3/perl/read/sqlite3_read.pl
#!/usr/bin/perl
#
#	sqlite3/perl/sqlite3_read.pl
#
#				May/19/2011
#
use DBI;
#
use	strict;
use	warnings;
#
use lib '/var/www/data_base/common/perl_common';
use sqlite3_manipulate;
# ---------------------------------------------------------
print "*** 開始 ***\n";
my $db_in = $ARGV[0];
#
#
#
my %dict_aa = sqlite3_manipulate::sqlite3_to_dict_proc ($db_in);
text_manipulate::dict_display_proc (%dict_aa);
#
print "*** 終了 ***\n";
# ---------------------------------------------------------

File: sqlite3/python/read/sqlite3_read.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	read/sqlite3_read.py
#					Sep/06/2016
#
# ------------------------------------------------------------------
import	sys
import	sqlite3
sys.path.append ('/var/www/data_base/common/python_common')
from sql_manipulate import sql_to_dict_proc
from text_manipulate import dict_display_proc
#
# ------------------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")
#
file_in=sys.argv[1]
conn = sqlite3.connect (file_in)
# conn.text_factory=str
conn.row_factory = sqlite3.Row
#
cursor = conn.cursor ()
#
dict_aa = sql_to_dict_proc (cursor)
#
cursor.close ()
conn.close ()
#
dict_display_proc (dict_aa)
#
sys.stderr.write ("*** 終了 ***\n")
# ------------------------------------------------------------------
